extend/custom-theme.md

```diff
@@ -104,3 +104,109 @@ app.Settings().SetTheme(&myTheme{})
 
 With these changes you can apply your own style, to make small tweaks or
 provide a completely custom looking application!
+
+### Workable Example
+```
```

Comment:
Let’s add some syntax highlighting to the code. It will be basically unreadable without.
```suggestion
```go
```